cmd/wallet-web/src/components/SkeletonLoader/CredentialPreviewSkeletonComponent.vue

```diff
@@ -5,7 +5,6 @@
 -->
 
 <script setup>
-import { defineProps } from 'vue';
```

Comment:
`defineProps` no longer need to be imported since they are defined as compiler macros and are available inside the `setup` script by default https://vuejs.org/api/sfc-script-setup.html#defineprops-defineemits
cc @HeidiHan0000

Reply:
Thanks for letting me know, will keep in mind next time!